### ISSUE TITLE
Final Steps of Deployment

### ISSUE THREAD

**Issue by Majovero** [Microsoft/mattercenter#430]
Just a a couple of quick Office Dev Noob questions. Going strictly by the Deployment Guide, I'm not sure what variables to change in "WebDashboardSPTemplate.html". I understand that I have to input the URL of the App created with the deployment script in two places, but are "matterdashboard" and "documentdashboard" both pointing to "https://serviceapp.azurewebsites.webdashboard.aspx"? See code below.

```
 $(document).ready(function () {
        "use strict";
        if ($("#webDashboardPage").length) {
            oWebDashboardSPGlobal.iFrame = $("#webDashboardPage")[0];
        }
        var sClientName = "", sSectionID = getParameterByName(sSection), sQueryString = "";
        if (sSectionID) {
            sQueryString = sQuestionMarkChar + sSection + sEqualChar + sSectionID + sAmpersandChar;
        }
        if(sSectionID==="1"){
            oWebDashboardSPGlobal.iFrame.src = "WEB_DASHBOARD_PAGE_URL/#/matterdashboard";
        }
        else{
            oWebDashboardSPGlobal.iFrame.src = "WEB_DASHBOARD_PAGE_URL/#/documentdashboard";
        }       
    })
```

Next question is in regards to the Web App Publishing. Do I upload the entire contents of the folder that was published to Apps for Office? Or is there some other type of file that needs to be uploaded. I went over the contents of the site hyperlinked in the Deployment Docs, but it only covers web deployment.  

Last question is in regards to creating a new page using the wiki template and naming it MatterCenterHome. Its very unclear to me what exactly is going on in this section of the guide. The scripts created a page called MatterCenterHome.aspx. Am I deleting the current one and creating a new one per the guide?

![image](https://cloud.githubusercontent.com/assets/16903695/18836442/1567f6c0-83ce-11e6-94a8-2759e9e29dd3.png)

I have these two files after all scripts are ran.

![image](https://cloud.githubusercontent.com/assets/16903695/18836495/48fe38e6-83ce-11e6-91f4-de7dc3eb2fba.png)

**Comment by WilsonReddy** [comment]
@Majovero Appendix A is to create new client. You need not to folloow Appendix A steps if you already specified your client information in MCDeploymentConfig.xlsx Client-Config tab. The deployment will create clients.  

Regarding publishing web site, you need to publishing only web site.

**Comment by Majovero** [comment]
Thanks @WilsonReddy. When you say publishing only website, I'm not sure I follow. I published only Microsoft.Legal.MatterCenter.Web. It outputs a folder full of website contents. Do I upload that whole folder to Sharepoint Office Apps? The last time I did any of this you could publish and get a .app file.

**Comment by WilsonReddy** [comment]
@Majovero You need to publish Microsoft.Legal.MatterCenter.Web to AzureWebsite.

I copied steps you need to perform for publishing website from https://blogs.msdn.microsoft.com/microsoftimagine/2015/07/20/welcome-to-visual-studio-2015-with-azure/ article.

![image](https://cloud.githubusercontent.com/assets/9116038/18847469/3b730740-83df-11e6-92fb-e328825ee85e.png)

**Comment by Majovero** [comment]
Thanks @WilsonReddy. I understand that process, I was just stumped on Step 4? 

![image](https://cloud.githubusercontent.com/assets/16903695/18853701/a6802e66-8414-11e6-9124-532af7b4c71b.png)

Do I just upload the entire published folder to Sharepoint Apps for Office.

**Comment by WilsonReddy** [comment]
@Majovero  Yes, you are correct. This is for work app.

**Comment by Majovero** [comment]
@WilsonReddy SharePoint Online does not allow you to upload a folder to Apps for Office. It has to be a single file. Which file created do I upload?

**Comment by amitabhm21** [comment]
@Majovero you have to upload the app manifest (Microsoft.Legal.MatterCenter_Office.xml) file

**Comment by Majovero** [comment]
Thanks @amitabhm21! I tried it before, but it didn't populate the GUID fields like regular apps do, so I figured it was something else. I appreciate that.

Matter Center is accessed from "https://mysite.sharepoint.com/SitePages/MatterCenterHome.aspx? Both my web apps also bring up the page below. Is this a known issue?

![image](https://cloud.githubusercontent.com/assets/16903695/18912316/c10ebbd0-854f-11e6-9062-4627d4e79d10.png)

**Comment by amitabhm21** [comment]
This is not a known issue. What do you mean both your webapps? As part of the matter center deployment one azure webapp is created which you should be able to open and browse. 

Can you conform that you get the matter center app loading properly when you browse to the webapp that was created  by the deployment? Also could you please attach the deployment log file ?

**Comment by Majovero** [comment]
The second app I referenced was the one I deployed out of VS per the deployment guide. I see that the certificate is there.

![image](https://cloud.githubusercontent.com/assets/16903695/19048895/315295c2-8976-11e6-9741-0fa2b07ff90a.png)

The script completed with no issues.

[MCDeploy20161003-053038.txt](https://github.com/Microsoft/mattercenter/files/506944/MCDeploy20161003-053038.txt)

**Comment by Majovero** [comment]
Thanks in advance for the help.Finally had some time to get back to this project. I have tried this with the most recent files. Still a no go. Same exact issue from a couple of months ago. I get no errors at all during the deployment. It's like the app is missing code, but how is that possible? I use VS 15 u3 with the latest pre-Req components. Thanks in advance for the help.

![empty_app](https://cloud.githubusercontent.com/assets/16903695/21527716/1863074a-ccfb-11e6-90e0-8088901d3aa8.JPG)

[Log.txt](https://github.com/Microsoft/mattercenter/files/675960/Log.txt)

**Comment by Majovero** [comment]
If possible could someone please take a screenshot of the file hierarchy for the web app? Even tho I have no deployment errors, I don't think the app files are being deployed correctly. @amitabhm21 @WilsonReddy

**Comment by JamesMonahan** [comment]
I get the same behavior.  I had some struggles with getting it to deploy but in the end succeeded.  I got through the remaining steps but I had to switch to the new Azure portal in Step 8 to get the Perms to display.  I Published the Microsoft.Legal.MatterCenter.Web in step 8.3.C and I get a "This web app has been successfully created". splash page.  I will try the process again but I think there is something missing in the web app.
